tests/unit/query_helpers/test_attachment_appender.py

```diff
@@ -141,6 +161,52 @@ def test_append_two_attachments_to_query(_test_yaml):
     assert output == expected
 
 
+def test_append_incomplete_yaml_query(_yaml_without_kind, _yaml_without_name):
```

Comment:
Should those fixtures be underscored? They return and are used.

Reply:
tbh I don't care. If you prefer, I'll change it now

Reply:
updated @onmete